index.html

```diff
@@ -8,17 +8,7 @@
 </head>
 
 <body>
-    <table class="table">

```

Comment:
Hello. I see that you removed this block of code. Please add it back, and change the header text (<th>) to the default given to you. Your table does not have a heading at the moment as well as an opening table tag